src/sentry/static/sentry/app/components/placeholder.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import space from 'app/styles/space';
+
+const Placeholder = styled(props => {
+  const {className} = props;
+  return <div className={className} />;
+})`
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  justify-content: center;
+
+  background-color: ${p => p.theme.placeholderBackground};
+  width: ${p => p.width};
+  height: ${p => p.height};
+  ${p => (p.shape == 'circle' ? 'border-radius: 100%;' : '')}
+  ${p =>
+    p.bottomGutter > 0 ? `margin-bottom: ${space(p.bottomGutter)};` : ''}
```

Comment:
My thought with the prop was that it could save us adding styled components in a few places. I can remove it though as using styled() will be more flexible.